support-frontend/app/services/PaymentAPIService.scala

```diff
@@ -100,12 +97,11 @@ class PaymentAPIService(wsClient: WSClient, paymentAPIUrl: String)(implicit ec:
   def executePaypalPayment(
     paymentJSON: JsObject,
     acquisitionData: JsValue,
-    queryStrings: Map[String, Seq[String]],
-    email: Option[String],
+    email: String,
```

Comment:
The existing code passes through arbitrary `queryStrings` to the Payment API. I can't see any use for this (the only query string it accepts is `mode=test`) so I've removed

Reply:
👍